side_length: 4.0
side_time: 1